feat: hiden auto_subscribe conf (#10381)
emqx/emqx: changes/ce/feat-10381.en.md

> Hide the auto_subscribe configuration item so that it can be modified later only through the HTTP API.

Reviewer: ```suggestion
Hide the `auto_subscribe` configuration items so that they can be modified later only through the HTTP API.
```
